Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:2.7-slim
```

Comment:
Something I've been wondering lately, is whether starting from `python` image really helps. If you're doing any python related `apt-get`ing later on -which you do- debian is downloading it's own version of python as well. So maybe you 're better of by directly using `debian` as base image. If you're to go with `python3` maybe that was a different case, but for python2.7 it's something to consider.